README.md

```diff
@@ -0,0 +1,2 @@
+# fpop
+`fp` stands for first principle calculation and `op` stands for operators. The abbreviation `fpop` stands for operators related to first principle calculation.
```

Comment:
first-principles

Reply:
Suggest add some basic background here, such as based on `dflow` (attached link). Furnish the words.